eslint/babel-eslint-plugin/package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-rule-composer": "^0.3.0"
   },
   "devDependencies": {
-    "eslint": "^7.5.0",
-    "lodash": "^4.17.20"
+    "clone-deep": "^4.0.1",
```

Comment:
clone-deep is quite bloated package. Maybe try [klona](https://www.npmjs.com/package/klona)?

Reply:
My strategy for choosing the `clone-deep` package was "Ctrl+F for `clone` in our `yarn.lock` file", so thanks for suggesting an alternative :joy:

Reply:
Oh it looks like `klona` requires Node.js >= 8, so until the Babel 8 release we cannot use it.